dali/pipeline/operators/crop/crop.h

```diff
@@ -162,7 +161,7 @@ class Crop : public Operator<Backend>, protected CropAttr {
   }
 
  protected:
-  Tensor<CPUBackend> input_ptrs_, input_strides_, output_offsets_;
+Tensor<CPUBackend> input_ptrs_, input_strides_, output_offsets_;
```

Comment:
Is `protected` here necessary? I mean, `Crop` doesn't really look like a class, which is designed to be inherited from (no virtual function, except those which are overridden)
Plus a minor indentation slip

Reply:
I don't know, I just made an unnecessary indentation change. I only wanted to add assert for number of dimensions in Crop, I do not want to rework it's API